GitUI/FindFilePredicateProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using GitCommands;
+
+namespace GitUI
+{
+    public interface IFindFilePredicateProvider
+    {
+        Func<string, bool> Get(string name, string workingDir);
```

Comment:
What does the parameter `name` contain? Is it a _file name_?

Reply:
It's pattern to match with file pathes. I rename it and add comment.